torch/csrc/jit/serialization/export.cpp

```diff
@@ -339,7 +339,7 @@ void EncoderBase::EncodeValueInfo(
           shape->mutable_dim(i)->set_dim_value(sizes[i].static_size());
         } else {
           if (symbol_dim_map_.find(sizes[i]) == symbol_dim_map_.end()) {
-            symbol_dim_map_[sizes[i]] = name + "_" + std::to_string(i);
+            symbol_dim_map_[sizes[i]] = "dim_" + std::to_string(i);
```

Comment:
Need to keep the name or some other way to ensure the dim_param is unique. dim_param with same string are expected to have same size at runtime.

Reply:
I see. What do you suggest for name? Since node name can be a number, previously I'd see examples like "2_0" where "2" was the debug name. So changing this to "2_dim_0" ?

Reply:
maybe include optype as well, like "Conv2_dim_0". the optype will be from `value->node()`. need to be careful about edge cases where `value` is graph input (not output from other node). 

not sure if each `value` would have other more displayable `name` available at that stage.